.circleci/config.yml

```diff
@@ -268,6 +271,123 @@ jobs:
+      #     test-apk-path: WordPress/build/outputs/apk/androidTest/vanilla/debug/org.wordpress.android-vanilla-debug-androidTest.apk
+      #     test-targets: package org.wordpress.android.ui.screenshots
+      #     devices: |
+      #       model=blueline,version=28,orientation=portrait
+      #       model=gts3lltevzw,version=28,orientation=default
+      #     # [Trial Limitation] Free plan I use to experiment with Firebase Test Lab has a quota of 10 tests/day
+      #     # locales: ar de_DE en_US es_ES fr_FR iw_IL in it_IT ja_JP ko_KR nl_NL pl_PL pt_BR ru_RU sr sv_SE th tr_TR vi zh_CN zh_TW
+      #     locales: en_US in
+      #     project: wpandroid-74b6c
+      #     timeout: 30m
+      #     num-flaky-test-attempts: 2
+      #     results-history-name: CircleCI WordPress Screenshots
+      # - android/save-gradle-cache
+
+      # BEGIN DEBUG STEP (While firebase-test is commented out)
+      - run:
+          name: Fake FTL Run
+          command: |
+            gcloud auth activate-service-account --key-file ".circleci/.firebase.secrets.json"
+            echo "INFO: Raw results root path is: [gs://test-lab-d1urt75q3y6wu-ha5s1mih364hi/2020-08-26_08:51:14.975190_umlG/]" >log.txt
+      # END DEBUG STEP
+
+      - run:
+          name: Download Raw Screenshots from Google Storage
+          command: |
+            TEST_BUCKET=$(cat log.txt | grep -o "gs://test\-lab\-.*/" | head -1)
+            bundle exec fastlane download_raw_screenshots bucket:"$TEST_BUCKET" phone:blueline-28 tenInch:gts3lltevzw-28
+      - persist_to_workspace:
+          root: .
+          paths:
+            - Gemfile
+            - Gemfile.lock
+            - fastlane/
+  promo-screenshots:
+    parameters:
+      post-to-slack:
+        description: Post to Slack when screenshots job has finished. SLACK_WEBHOOK ENV variable must be set.
+        type: boolean
+        default: true
+    macos:
+      xcode: 11.6.0
+    working_directory: /tmp/workspace
+    steps:
+      - attach_workspace:
+          at: .
+      - restore_cache:
+          # ImageMagick is used by the 'rmagick' gem, so cache invalidation of ImageMagick should depend on version of rmagick from Gemfile.lock
+          keys:
+            - &homebrew-cache-key v2-brew-imagemagick-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Brew Install ImageMagick
+          command: |
+            brew install pkg-config imagemagick
+            brew cleanup
+      - save_cache:
+          paths:
+            - ~/Library/Caches/Homebrew
+          key: *homebrew-cache-key
+      - run:
+          name: Bundle install with rmagick
+          command: |
+            # Activate 'screenshots' in bundle config options (remove it from 'without' in case it's set here)
+            bundle config --local --delete without
+            bundle config --local with screenshots
+            bundle install --path vendor/bundle
```

Comment:
I tried to use the `bundle-install/bundle-install` orb command here instead of calling `bundle install --path vendor/bundle` myself, but there's a cache issue with this orb command when we have symlinks ([see this build for example where I tried that](https://app.circleci.com/pipelines/github/AliSoftware/WordPress-Android/115/workflows/4540300b-42b8-4eec-a2bc-dcd318ffabde/jobs/302)). This is apparently a known limitation of CircleCI that sometimes happen in some situations and when the folder contains symlinks, with no clean workaround for it.